charts/rancher-monitoring/v0.0.4/charts/exporter-node-windows/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name:   expose-node-metrics-windows
+  labels:
+    app: {{ template "app.name" . }}
+    chart: {{ template "app.version" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    os: windows
+    {{ .Values.apiGroup }}: "true"
```

Comment:
Why do we need this kind of label? It is not that common in community charts I've seen.

Reply:
removed